cpp/clang/.clang-format

```diff
@@ -39,6 +39,7 @@ BraceWrapping:
   SplitEmptyFunction: false
   SplitEmptyRecord: false
   SplitEmptyNamespace: false
+BracedInitializerIndentWidth: 2
```

Comment:
Maybe it would be better to use 4 spaces here as it was done before and as we use in regular indents (I mean same spaces count in initializers and other places)?

Reply:
agree

Reply:
🤝